benchmarks/benchmark_header.h

```diff
@@ -52,7 +52,12 @@ std::vector<std::string> split_string(const std::string& str) {
   auto result = std::vector<std::string>{};
   auto ss = std::stringstream{str};
   for (std::string line; std::getline(ss, line, '\n');) {
-    result.push_back(line);
+    std::string_view view = line;
+    // Some parsers like boost/url will refuse to parse a URL with trailing
```

Comment:
Boost and curl crash if there is a space in the URL too...

Reply:
"Boost and curl crash if there is a space in the URL too..." 

They will refuse to parse the URL, and that's because they follow a different standard. Evidently, this means that they are solving a slightly easier problem.

For the leading and trailing spaces, there is a lot of them and we can argue that they may have been the result of our URL processing, so it is fair to remove them (and quite cheap).